contracts/peripheral/IncurDebt.sol

```diff
@@ -242,17 +240,14 @@ contract IncurDebt is OlympusAccessControlledV2, IIncurDebt {
     /// @param _borrower the account to seize collateral
     function seize(address _borrower) external override onlyGovernor isBorrower(_borrower) {
         (uint256 seizedCollateral, uint256 paidDebt) = _repay(_borrower);
+        borrowers[_borrower].collateralInGOHM = 0;
 
-        clearBorrower(_borrower);
-
-        IERC20(sOHM).safeApprove(staking, seizedCollateral);
-        IStaking(staking).unstake(address(this), seizedCollateral, false, true);
-        IOHM(OHM).burn(seizedCollateral);
+        IgOHM(gOHM).burn(address(this), seizedCollateral);
```

Comment:
Only staking contract can burn ohm
Unstake borrowers sohm to ohm and burn

Reply:
done